frontend/src/pull-card/flags.tsx

```diff
@@ -3,13 +3,14 @@ import { actionMessage } from '../utils';
 import { Link, Box, useStyleConfig } from "@chakra-ui/react"
 import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-import { faWarning, faMinusCircle, faEye, faEyeSlash } from '@fortawesome/free-solid-svg-icons'
+import { faWarning, faMinusCircle, faEye, faEyeSlash, faIcicles } from '@fortawesome/free-solid-svg-icons'
```

Comment:
I think the snowflake might be more intuitive

Reply:
Great call! Fixed!